extra-cutefish/cutefish-wallpapers/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=cutefish-wallpapers
+PKGSEC=Cutefish
+PKGDES="System wallpapers of Cutefish DE"
+
+ABHOST=noarch
```

Comment:
No line ending.

Reply:
I accidentally deleted it. Will fix soon.